src/Microsoft.Data/BaseColumn.Computations.cs

```diff
@@ -12,67 +12,165 @@ namespace Microsoft.Data
+        /// </summary>
         public virtual void Abs()
         {
             throw new NotImplementedException();
         }
 
+        /// <summary>
+        /// Returns whether all the elements are True
+        /// </summary>
         public virtual bool All()
         {
             throw new NotImplementedException();
         }
 
+        /// <summary>
+        /// Returns whether any element is True
+        /// </summary>
         public virtual bool Any()
         {
             throw new NotImplementedException();
         }
 
+        /// <summary>
+        /// Updates each element with its cumulative maximum
+        /// </summary>
         public virtual void CumulativeMax()
         {
             throw new NotImplementedException();
         }
 
+        /// <summary>
+        /// Updates column values at rowIndices with its cumulative rowIndices maximum
+        /// </summary>
+        public virtual void CumulativeMax(IEnumerable<long> rowIndices)
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <summary>
+        /// Updates each element with its cumulative minimum
+        /// </summary>
         public virtual void CumulativeMin()
         {
             throw new NotImplementedException();
         }
 
+        /// <summary>
+        /// Updates column values at rowIndices with its cumulative rowIndices minimum
+        /// </summary>
+        public virtual void CumulativeMin(IEnumerable<long> rowIndices)
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <summary>
+        /// Updates each element with its cumulative product
+        /// </summary>
         public virtual void CumulativeProduct()
         {
             throw new NotImplementedException();
         }
 
+        /// <summary>
+        /// Updates column values at rowIndices with its cumulative rowIndices product
+        /// </summary>
+        public virtual void CumulativeProduct(IEnumerable<long> rowIndices)
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <summary>
+        /// Updates each element with its cumulative sum
+        /// </summary>
         public virtual void CumulativeSum()
         {
             throw new NotImplementedException();
         }
 
+        /// <summary>
+        /// Updates column values at rowIndices with its cumulative rowIndices sum
+        /// </summary>
+        public virtual void CumulativeSum(IEnumerable<long> rowIndices)
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <summary>
+        /// Returns the maximum of the values in the column
+        /// </summary>
         public virtual object Max()
         {
             throw new NotImplementedException();
         }
 
+        /// <summary>
+        /// Returns the maximum of the values at rowIndices
+        /// </summary>
+        public virtual object Max(IEnumerable<long> rows)
```

Comment:
The parameter here is still called `rows`, which is inconsistent with the summary comment and the methods above.